plugins/TwoFactorAuth/SystemSettings.php

```diff
@@ -29,11 +29,17 @@ protected function init()
 
     private function createRequire2FA()
     {
-        return $this->makeSetting('twoFactorAuthRequired', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
-            $field->title = 'Require two-factor authentication for everyone';
-            $field->description = 'When enabled, every user has to enable two factor authentication.';
+        $setting = $this->makeSetting('twoFactorAuthRequired', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+            $field->title = Piwik::translate('TwoFactorAuth_RequireTwoFAForAll');
+            $field->description = Piwik::translate('TwoFactorAuth_RequireTwoFAForAllInformation');
             $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
         });
+
+        $twoFa = StaticContainer::get(TwoFactorAuthentication::class);
+        $isWritable = $twoFa->isUserUsingTwoFactorAuthentication(Piwik::getCurrentUserLogin());
+        $setting->setIsWritableByCurrentUser($isWritable);
```

Comment:
this requires some more use statements in the head:
```
use Piwik\Plugins\TwoFactorAuth\TwoFactorAuthentication;
use Piwik\Container\StaticContainer;
```

Reply:
Done. Thanks :)